utils/installer/lv-config.example-no-ts.lua

```diff
@@ -16,7 +16,25 @@ O.colorscheme = "spacegray"
 O.auto_close_tree = 0
 O.default_options.wrap = true
 O.default_options.timeoutlen = 100
-O.leader_key = " "
+-- keymappings
+O.keys.leader_key = "space"
+-- you can override all the default keymappings (those that aren't prefixed with <leader>)
+O.keys.normal_mode = {
```

Comment:
can you comment these lines :pleading_face:

Reply:
Do you mean comment-out the empty list? I was just thinking people might miss it. But it's also a bit risky if someone didn't notice that it'll actually completely overwrite the default list for that mode..

Reply:
yeah , pretty much comment out all the keybinding stuff except `O.keys.leader_key = "space"`

so that it informs users but doesn't change the default behavior